libc/include/llvm-libc-types/struct_epoll_event.h

```diff
@@ -0,0 +1,19 @@
+//===-- Definition of epoll_event type ------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef __LLVM_LIBC_TYPES_EPOLL_EVENT_H__
+#define __LLVM_LIBC_TYPES_EPOLL_EVENT_H__
+
+#include <llvm-libc-types/struct_epoll_data.h>
```

Comment:
Will changing this to `#include "struct_epoll_data.h"` make direct inclusion from `libc/src` folder works for both overlay and full build mode?

Reply:
No, since we'd also have to change the headers for `sigset_t` and `timespec`. Additionally, using a non-system header for those two in overlay mode causes this error in the tests: `/usr/include/x86_64-linux-gnu/bits/types/sigset_t.h:7:20: error: typedef redefinition with different types ('__sigset_t' vs 'struct sigset_t')`.

It might be possible to move to the quotes in future, but that's beyond the scope of this patch.